tests/models/clip/test_modeling_clip.py

```diff
@@ -353,6 +353,7 @@ class CLIPTextModelTest(ModelTesterMixin, unittest.TestCase):
     fx_compatible = True
     test_pruning = False
     test_head_masking = False
+    model_split_percents = [0.5, 0.8, 0.9]
```

Comment:
the changes from this to the end is `0.7` is not working.
(`0.5` is not used in the `test_model_parallelism` as we change it to `self.model_split_percents[1:]` in this PR)